diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -667,6 +738,18 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
         override fun fix() {
             node.appendNewlineMergingWhiteSpace(null, node.findChildByType(EQ)?.treeNext)
         }
+
+        override fun unFix() {
+            node.findChildAfter(EQ, WHITE_SPACE)?.let {
+                if (it.textContains('\n')) {
+                    it.nextSibling()?.let { it2 ->
```

Comment:
suggestion to use some meaningful name instead of `it2` and `it` here

Reply:
Done